src/app/app.routes.ts

```diff
@@ -1,26 +1,31 @@
-import { Routes } from '@angular/router';
-import { AuthGuard } from './guard/auth.guard';
-import { HomeComponent } from './products/home/home.component';
-import { EmptyCartComponent } from './products/empty-cart/empty-cart.component';
-import { DetailsComponent } from './products/details/details.component';
-import { LoginComponent } from './auth/login/login.component';
-import { CheckoutComponent } from './checkout/checkout.component';
-import { SignupComponent } from './auth/signup/signup.component';
+import {Routes} from '@angular/router';
+import {authGuard} from './guard/auth.guard';
 
 export const routes: Routes = [
-  { path: '', component: LoginComponent },
-  { path: 'signup', component: SignupComponent },
-  { path: 'home', component: HomeComponent,canActivate: [AuthGuard] },
+  {path: '', loadComponent: () => import('./auth/login/login.component').then((m) => m.LoginComponent)},
```

Comment:
It will be nice to prevent logged in users from accessing signup or login pages. Since they are already logged let's redirect them to the right place